### PR TITLE
PS-8394 8.0.30-22 release notes and release note tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
A  docs/release-notes/8.0.30-22.md
M  docs/release-notes/release-notes_index.md
M  mkdocs-base.yml

### CODE REVIEW

File: docs/release-notes/8.0.30-22.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Bug fixes
+
+* [PS-8204](<https://jira.percona.com/browse/PS-8204/>): A fix for when the `audit_log_format` was set to XML and the logged queries were truncated after a newline character.
+
+* [PS-8246](https://jira.percona.com/browse/PS-8246): Adapted the Encryption UDF MTR test cases to OpenSL 3.0.x. The tests failed because OpenSSL 3.0.x has a different default set of hash functions and PEM reading and writing routines generate different error messages.
+
+* [PS-8351](https://jira.percona.com/browse/PS-8351): The `SHOW ENGINE INNODB STATUS` would output unreadable UTF characters. 
+
+* [PS_8364](https://jira.percona.com/browse/PS-8364): When using Docker, the data directory for ICU regular expressions was missing.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0.30-22.md]: PS_8364 -> PS-8364

Comment (patrickbirch) [reply docs/release-notes/8.0.30-22.md]: Fixed.

**Review by george-lorch — APPROVED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Comment by alina-derkach-oaza** [comment]
I closed this PR due to multiple commits. New PR https://github.com/percona/psmysql-docs/pull/47

**Pull request closed without merging**